Import css/filter-effects WPT.

mattwoodrow opened the pull request:
#### e8b30e3a7c6b93d6dae92dd78fea0e45a69c0cae
<pre>
Import css/filter-effects WPT.
<a href="https://bugs.webkit.org/show_bug.cgi?id=261549">https://bugs.webkit.org/show_bug.cgi?id=261549</a>
&lt;rdar://115472044&gt;

Reviewed by Tim Nguyen.

Import of rev 9593290ad1bd621f74c697c7cc347348af2de32a.

Note that this includes rewriting backdrop-filter to -webkit-backdrop-filter (which wasn&apos;t done on the previous
import somehow). This makes a lot of tests pass, even though they don&apos;t pass on WPT.fyi.

I&apos;ll reimport the tests again once we unprefix to remove the rewriting.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/animation/backdrop-filter-interpolation-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/animation/color-interpolation-filters-no-interpolation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/animation/color-interpolation-filters-no-interpolation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/animation/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-basic-background-color-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-basic-background-color.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-basic-opacity-2.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-basic-opacity.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-basic.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-rect-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-rect-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-rect.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-rounded-clip.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clipped.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-containing-block.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-edge-behavior.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-edge-clipping.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-edge-pixels.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-feimage-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-fixed-clip.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-invalid-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-invalid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-isolation-fixed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-isolation-isolate.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-isolation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-opacity-rounded-clip.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-paint-order.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-filter-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-filter-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-filter.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-opacity.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-will-change-opacity-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-will-change-opacity.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-reference-filter-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-reference-filter.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-root-element.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-svg-background-image-blur-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-svg-background-image-blur.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-svg-foreignObject.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-update.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-zero-size.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-brightness.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-contrast.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-grayscale-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-grayscale-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-grayscale-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-hue-rotate.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-invert.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-opacity.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-saturate.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-sepia.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/background-image-blur-repaint-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/background-image-blur-repaint.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/crashtests/broken-reference-crash-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/crashtests/felighting-display-none-mutation-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-blur-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-blur-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-blur.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-brightness-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-brightness-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-brightness.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-combined-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-combined-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-combined.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-contrast-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-contrast-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-contrast.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-drop-shadow-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-drop-shadow-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-drop-shadow.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-grayscale-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-grayscale-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-grayscale.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-hue-rotate-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-hue-rotate-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-hue-rotate.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-invert-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-invert-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-invert.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-opacity-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-opacity-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-opacity.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-saturate-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-saturate-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-saturate.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-sepia-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-sepia-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-sepia.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-filters-animation-blur-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-filters-animation-blur-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-filters-animation-blur.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-filters-animation-combined-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/effect-reference-after-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/effect-reference-delete-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/effect-reference-feimage-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/effect-reference-feimage-002-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/effect-reference-feimage-003-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/effect-reference-feimage-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/effect-reference-feimage-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/effect-reference-rename-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/effect-reference-source-alpha-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/fecolormatrix-type.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/feimage-circular-reference-foreign-object-crash.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/feimage-reference-foreign-object-crash.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-cb-abspos-inline-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-cb-abspos-inline-001-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-cb-abspos-inline-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-cb-abspos-inline-002-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-cb-abspos-inline-003-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-cb-abspos-inline-003-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-chained-url-url-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-chained-url-url-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-contrast-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-contrast-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-contrast-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-effect-remove-unattached-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-effect-remove-unattached.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-001-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-002-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-003-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-004-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-005-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-006-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-006.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-007-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-007.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-008-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-008.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-conic-gradient-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-conic-gradient-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-conic-gradient.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-linear-gradient-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-linear-gradient-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-linear-gradient.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-radial-gradient-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-radial-gradient-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-radial-gradient.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-repeating-conic-gradient-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-repeating-conic-gradient-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-repeating-conic-gradient.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-repeating-linear-gradient-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-repeating-linear-gradient-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-repeating-linear-gradient.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-repeating-radial-gradient-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-repeating-radial-gradient-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-repeating-radial-gradient.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/resources/green-transparent-100x100.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/resources/green-transparent-200x200.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/resources/green-transparent-20x10.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/resources/green-transparent-80x80.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/resources/w3c-import.log: Copied from LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/crashtests/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-hue_rotate-001-test.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-invalid-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-invalid-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-invalid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-calc-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-calc-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-transformed-child-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-transformed-child-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-transformed-composited-child-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-transformed-composited-child-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-units-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-units-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-saturate-001-test.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-scale-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-scaling-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-subregion-01.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-svg-background-image-blur-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-svg-background-image-blur.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-turbulence-invalid-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filters-drop-shadow-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filters-drop-shadow-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filters-test-brightness-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filters-test-brightness-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filters-test-brightness-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/fixed-pos-filter-clip-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/fixed-pos-filter-clip-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/fixed-pos-filter-clip-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/fixed-pos-filter-clip-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/fixed-pos-filter-clip-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/repaint-added-backdrop-filter-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/repaint-added-backdrop-filter.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-empty-container-with-filter-content-added.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-external-filter-resource-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-external-filter-resource.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-filter-vs-clip-path.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-image-root-filter-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-image-root-filter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-image-root-filter.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-multiple-filter-functions.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-mutation-filter-used-by-mask-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-mutation-filter-used-by-mask.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-relative-urls-002-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-sourcegraphic-currentcolor-dynamic-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-sourcegraphic-invalidation-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-unused-filter-on-clippath-mutated-crash.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-feblend-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fecomponenttransfer-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fecomposite-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-feconvolvematrix-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fediffuselighting-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fediffuselighting-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fediffuselighting-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fediffuselighting-dynamic.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fedisplacementmap-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fedropshadow-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-feflood-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fegaussianblur-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-femorphology-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-feoffset-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fespecularlighting-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fespecularlighting-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fespecularlighting-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fetile-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/will-change-blur-filter-under-clip-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/will-change-blur-filter-under-clip.html: Added.

Canonical link: <a href="https://commits.webkit.org/268004@main">https://commits.webkit.org/268004@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/ef7a5e865d6f418468b08a3eba1bb27328272817

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/18272 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/18607 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/19184 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/20112 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/17101 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/21903 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/18760 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/19036 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/18494 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/18702 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/15911 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/20995 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/15925 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/16664 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/23160 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/16944 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/16835 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/21050 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/17400 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/14758 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/16497 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/16503 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/4371 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/20860 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/17244 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/animation/backdrop-filter-interpolation-001.html:
```diff
@@ -2,15 +2,15 @@
     <style>
       .parent {
-        backdrop-filter: hue-rotate(30deg);
+        -webkit--webkit-backdrop-filter: hue-rotate(30deg);
```
This seems wrong. cc @gsnedders

mattwoodrow replied:
Whoops, that was me. The script didn't covert these 4 animation tests (since the properties are just JS strings), so I did them with search and replace.

I am impressed that you looked enough to find this :)

mattwoodrow committed:
Import css/filter-effects WPT.
https://bugs.webkit.org/show_bug.cgi?id=261549
<rdar://115472044>

Reviewed by Tim Nguyen.

Import of rev 9593290ad1bd621f74c697c7cc347348af2de32a.

Note that this includes rewriting backdrop-filter to -webkit-backdrop-filter (which wasn't done on the previous
import somehow). This makes a lot of tests pass, even though they don't pass on WPT.fyi.

I'll reimport the tests again once we unprefix to remove the rewriting.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/animation/backdrop-filter-interpolation-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/animation/color-interpolation-filters-no-interpolation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/animation/color-interpolation-filters-no-interpolation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/animation/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-basic-background-color-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-basic-background-color.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-basic-opacity-2.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-basic-opacity.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-basic.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-rect-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-rect-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-rect.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-rounded-clip.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clipped.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-containing-block.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-edge-behavior.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-edge-clipping.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-edge-pixels.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-feimage-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-fixed-clip.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-invalid-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-invalid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-isolation-fixed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-isolation-isolate.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-isolation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-opacity-rounded-clip.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-paint-order.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-filter-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-filter-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-filter.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-opacity.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-will-change-opacity-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-will-change-opacity.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-reference-filter-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-reference-filter.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-root-element.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-svg-background-image-blur-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-svg-background-image-blur.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-svg-foreignObject.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-update.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-zero-size.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-brightness.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-contrast.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-grayscale-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-grayscale-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-grayscale-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-hue-rotate.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-invert.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-opacity.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-saturate.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filters-sepia.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/background-image-blur-repaint-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/background-image-blur-repaint.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/crashtests/broken-reference-crash-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/crashtests/felighting-display-none-mutation-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-blur-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-blur-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-blur.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-brightness-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-brightness-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-brightness.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-combined-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-combined-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-combined.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-contrast-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-contrast-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-contrast.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-drop-shadow-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-drop-shadow-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-drop-shadow.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-grayscale-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-grayscale-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-grayscale.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-hue-rotate-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-hue-rotate-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-hue-rotate.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-invert-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-invert-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-invert.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-opacity-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-opacity-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-opacity.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-saturate-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-saturate-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-saturate.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-sepia-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-sepia-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filters-animation-sepia.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-filters-animation-blur-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-filters-animation-blur-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-filters-animation-blur.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-filters-animation-combined-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/effect-reference-after-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/effect-reference-delete-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/effect-reference-feimage-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/effect-reference-feimage-002-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/effect-reference-feimage-003-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/effect-reference-feimage-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/effect-reference-feimage-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/effect-reference-rename-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/effect-reference-source-alpha-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/fecolormatrix-type.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/feimage-circular-reference-foreign-object-crash.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/feimage-reference-foreign-object-crash.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-cb-abspos-inline-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-cb-abspos-inline-001-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-cb-abspos-inline-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-cb-abspos-inline-002-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-cb-abspos-inline-003-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-cb-abspos-inline-003-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-chained-url-url-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-chained-url-url-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-contrast-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-contrast-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-contrast-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-effect-remove-unattached-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-effect-remove-unattached.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-001-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-002-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-003-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-004-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-005-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-006-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-006.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-007-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-007.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-008-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-008.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-conic-gradient-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-conic-gradient-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-conic-gradient.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-linear-gradient-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-linear-gradient-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-linear-gradient.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-radial-gradient-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-radial-gradient-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-radial-gradient.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-repeating-conic-gradient-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-repeating-conic-gradient-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-repeating-conic-gradient.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-repeating-linear-gradient-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-repeating-linear-gradient-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-repeating-linear-gradient.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-repeating-radial-gradient-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-repeating-radial-gradient-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/filter-function-repeating-radial-gradient.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/resources/green-transparent-100x100.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/resources/green-transparent-200x200.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/resources/green-transparent-20x10.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/resources/green-transparent-80x80.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/resources/w3c-import.log: Copied from LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/crashtests/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-function/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-hue_rotate-001-test.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-invalid-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-invalid-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-invalid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-calc-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-calc-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-transformed-child-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-transformed-child-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-transformed-composited-child-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-transformed-composited-child-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-units-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-units-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-saturate-001-test.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-scale-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-scaling-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-subregion-01.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-svg-background-image-blur-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-svg-background-image-blur.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-turbulence-invalid-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filters-drop-shadow-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filters-drop-shadow-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filters-test-brightness-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filters-test-brightness-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filters-test-brightness-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/fixed-pos-filter-clip-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/fixed-pos-filter-clip-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/fixed-pos-filter-clip-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/fixed-pos-filter-clip-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/fixed-pos-filter-clip-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/repaint-added-backdrop-filter-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/repaint-added-backdrop-filter.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-empty-container-with-filter-content-added.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-external-filter-resource-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-external-filter-resource.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-filter-vs-clip-path.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-image-root-filter-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-image-root-filter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-image-root-filter.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-multiple-filter-functions.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-mutation-filter-used-by-mask-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-mutation-filter-used-by-mask.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-relative-urls-002-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-sourcegraphic-currentcolor-dynamic-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-sourcegraphic-invalidation-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-unused-filter-on-clippath-mutated-crash.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-feblend-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fecomponenttransfer-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fecomposite-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-feconvolvematrix-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fediffuselighting-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fediffuselighting-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fediffuselighting-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fediffuselighting-dynamic.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fedisplacementmap-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fedropshadow-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-feflood-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fegaussianblur-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-femorphology-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-feoffset-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fespecularlighting-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fespecularlighting-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fespecularlighting-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/tainting-fetile-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/will-change-blur-filter-under-clip-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/will-change-blur-filter-under-clip.html: Added.

Canonical link: https://commits.webkit.org/268004@main
